editor/src/clj/editor/code/view.clj

```diff
@@ -3688,10 +3645,134 @@
     ;; Start repaint timer.
     (ui/timer-start! repainter)
     ;; Initial draw
-    (ui/run-later (repaint-view! view-node 0 {:cursor-visible true})
+    (ui/run-later (repaint-view! view-node 0 {:cursor-visible true :editable editable})
                   (ui/run-later (slog/smoke-log "code-view-visible")))
     view-node))
 
+(def ^:private fundamental-read-only-handlers
```

Comment:
This is the "read-only" `defhandlers` shared with the Console view from earlier expressed as data. Both the code editor and the Console view now call the `register-fundamental-read-only-handlers!` function to register these handlers with the respective views.